pom.xml

```diff
@@ -4,7 +4,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>io.github.woodser</groupId>
 	<artifactId>monero-java</artifactId>
-	<version>0.8.33</version>
+	<version>0.8.33x</version>
```

Comment:
Any reason to change the version number now? Normally I bump it just before release.